### PR TITLE
[MB-7053] Add domestic shorthaul support to service items calculator

### PR DESCRIPTION
**Pull request by hannaliebl**
## Description

Add pricing details for Domestic shorthaul.

## Reviewer Notes

I'm fairly sure `PriceRateOrFactor` and `EscalationCompounded` params aren't implemented yet, so the baseline shorthaul price and price escalation factors are blank for now.

## Setup

`make db_dev_e2e_populate`
`make server_run`
`make client_run`

## Code Review Verification Steps

TIO role -> PARAMS move locator.

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-7053) for this change

## Screenshots

Screen Shot 2021-03-24 at 7 36 29 PM

### CODE REVIEW

File: src/components/Office/ServiceItemCalculations/helpers.js [hunk 1]
```diff
@@ -54,6 +54,20 @@ const mileage = (params) => {
   return calculation(value, label, detail);
 };
 
+const mileageZip5 = (params) => {
```
Comment (hannaliebl) [inline src/components/Office/ServiceItemCalculations/helpers.js:57]: I can also see a world where the mileage function just handles both cases. Thoughts?

File: src/components/Office/ServiceItemCalculations/helpers.js [hunk 2]
```diff
@@ -72,6 +86,24 @@ const baselineLinehaulPrice = (params) => {
   return calculation(value, label, detail1, detail2, detail3);
 };
 
+const baselineShorthaulPrice = (params) => {
```
Comment (hannaliebl) [inline src/components/Office/ServiceItemCalculations/helpers.js:89]: This is essentially the same as the function above it, and I could see an argument for just having one...uh, "haul" function to handle both short and linehaul? For now I kept them separate, also to hopefully ease potential merge conflicts, but that might be premature, and I'm happy to just make one that accepts some parameters for short or line hauls.

**Review by donaldthai — APPROVED** [review]
Looks good to me. I think we just need to add the test case in the `helper.test.js` file.

**Comment by donaldthai** [comment]
Let's add a storybook thing too for this service item. I did it this way: https://github.com/transcom/mymove/pull/6220/files#diff-fe115b2dcdec206cdac17f841c58e17b8366dddfb04dbd1a2bbedfcdaf787f03R32

**Comment by hannaliebl** [comment]
> Looks good to me. I think we just need to add the test case in the `helper.test.js` file.

I just realized JP added it already 🎉

File: src/components/Office/ServiceItemCalculations/helpers.js (hunk 1 above)
Comment (jenniferlynparsons) [reply src/components/Office/ServiceItemCalculations/helpers.js:57]: I can see that, too. They're really not very different. We can make that a refinement for next week.

**Review by jenniferlynparsons — APPROVED** [review]
This looks good. Merging in the main base pricing branch should resolve the failing tests and then this is good to go. Keep your ZIP5 function, I removed it from mine. 😄

**Pull request merged**